### ISSUE TITLE
Add `minimum-tls-version` and `requires-forward-secrecy` options to `App.accessRule`

### ISSUE THREAD

**Issue by martijnwalraven** [meteor/meteor#6554]
Earlier versions of Cordova disabled App Transport Security on iOS entirely (setting `NSAllowsArbitraryLoads`). But the latest versions use the existing access rules to allow HTTP for specific domains (setting `NSExceptionAllowsInsecureHTTPLoads` for those domains). These domains still need to meet the default ATS requirements for SSL however (which means using TLS v1.2 and supporting forward secrecy).

Cordova has recently added new access options specifically to weaken these requirements (`minimum-tls-version` and `requires-forward-secrecy`), so we should also add these to our `App.accessRule` options.

See the [App Transport Security docs](https://developer.apple.com/library/ios/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html#//apple_ref/doc/uid/TP40009251-SW33) and the [Cordova whitelisting docs](https://cordova.apache.org/docs/en/dev/guide/appdev/whitelist/).

**Comment by danlg** [comment]
@martijnwalraven does that mean that there is no workaround at the moment ? I cannot remove SSL on production app...
if we add the below config in ```cordova-build-override/config.xml```  would that work   or does it require some meteor changes ?
```
<access origin="https://*.mydomain.com" minimum-tls-version="TLSv1.2" requires-forward-secrecy="false" />
  
```

**Comment by martijnwalraven** [comment]
I would generally avoid using the `cordova-build-override` mechanism, because it simply replaces files like `config.xml` and thus overwrites Cordova platform defaults and changes made by plugins.

If this is just a temporary workaround for a production build, I would suggest editing `Info.plist` directly (can also be done by opening the Xcode project in the build directory).

Is there any way you can enable support for forward secrecy on your servers? If you're already on TLS v1.2, that should be the only change required to meet the default ATS requirements.

**Comment by martijnwalraven** [comment]
Just pushed a commit that would allow the following in `mobile-config.js`:
`App.accessRule("https://www.domain.com", { "minimum-tls-version": "TLSv1.0", "requires-forward-secrecy": false });`

**Comment by benjamn** [comment]
Fixed in [`1.3-rc.6`](https://github.com/meteor/meteor/issues/6266#issuecomment-199903292).
